src/modules/Packages.rb

```diff
@@ -18,6 +18,10 @@ class PackagesClass < Module
     RESOLVABLE_TYPES = [:product, :patch, :package, :pattern, :language]
     # Minimum set of packages required to enable VNC server
     VNC_BASE_PACKAGES = ["xorg-x11", "xorg-x11-Xvnc", "xorg-x11-fonts", "xinetd"]
+    # Additional packages needed to run second stage in graphical mode
+    # Use exact package name (i.e. libyui-qt6 instead of libyui-qt) to prevent
+    # the UI from asking for confirmation
+    AUTOYAST_X11_PACKAGES = ["libyui-qt6", "yast2-x11"]
```

Comment:
I originally assumed the popup was being raised somewhere else, in a lower layer. But thinking it twice, it's must be here, in this same module. So I will look for the code raising it and see if I can avoid it in general during autoinstallation (although that's a change in the behavior).